background.js

```diff
@@ -40,3 +33,13 @@ chrome.tabs.onActivated.addListener(() => {
     });
   });
 });
+
+var id = 0;
+chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
+  id = tabId;
+});
+
+chrome.tabs.onRemoved.addListener(function (tabId, removeInfo) {
```

Comment:
Shouldn't onRemoved trigger a cleanup from local store?

Reply:
![image](https://user-images.githubusercontent.com/49064703/95539768-df77fc80-0a0c-11eb-90b6-e4e78fd7b853.png)
I think this is what you wanted i have fixed the issue by removing and consoling the local storage when OnRemoved is triggered when the tab exists.